admin-dev/themes/new-theme/js/pages/product/edit/product-options-manager.ts

```diff
@@ -79,6 +80,23 @@ export default class ProductOptionsManager {
     });
   }
 
+  private initProductVisibilityList(): void {
+    const defaultSelectedInput = document.querySelector(`${ProductMap.options.visibilityRadio}:checked`) as HTMLInputElement;
+    const descriptionField = document.querySelector('.js-description') as HTMLDivElement;
```

Comment:
Using a generic like `document.querySelector<HTMLDivElement>('.js-description')` would be safer and cleaner, and check if it exist right after

Sometime in our paradigm, we're tempted to cast the type as anyway the HTML is not generated by any JS lib meaning that we've no control over it, but when we can because the code part is pretty small, let's use the cleaner method!